fhir-server/liberty-config/config/default/fhir-server-config.json

```diff
@@ -8,15 +8,29 @@
             "conditionalDeleteMaxNumber": 10,
             "serverRegistryResourceProviderEnabled": true
         },
-        "authFilter": {
-            "enabled": false,
-            "authorizedClientCertClientCN": "",
-            "authorizedClientCertIssuerOU": ""
-        },
-        "oauth": {
-            "regUrl": "https://<host>:9443/oauth2/endpoint/oauth2-provider/registration",
-            "authUrl": "https://<host>:9443/oauth2/endpoint/oauth2-provider/authorize",
-            "tokenUrl": "https://<host>:9443/oauth2/endpoint/oauth2-provider/token"
+        "security": {
+            "cors": true,
```

Comment:
If we require at least one auth mechanism to be selected, maybe adding a comment here to highlight it in the security block/json object

Reply:
Remember, this is the "advertised" security capabilities (shows up in our CapabilityStatement), which doesn't necessarily need to match the real security used from the liberty config (e.g. server.xml).
For example, even with issue #1020 unresolved, its possible to use basic auth on the server but to add that auth via some proxy so that it appears "open" to end users.
Thus, I think its allowed to advertise no security mechanism.